### ISSUE TITLE
FamilyInstance.SetRotation crashes at angle 270

### ISSUE THREAD

**Issue by ghost** [DynamoDS/DynamoRevit#992]
## Dynamo version

1.0.0 (issue also occurs in 0.9.2)
## Revit version

Revit 2016
## Operating system

Windows 7
## What did you do?

Select model element (placed family) => FamilyInstance.SetRotation with a rotation angle of 270
## What did you expect to see?

rotated family
## What did you see instead?

revit and dynamo are totally hosed, hard crash

**Comment by kronz** [comment]
@AndrewHeumann Yep, we're on it.  Here's the pull request, which should also address lots of similar set parameter issues.
https://github.com/DynamoDS/DynamoRevit/pull/991

**Comment by johnpierson** [comment]
Closing this out - the thread indicates this was addressed, and it has had no further activity since.

If you still see this behaviour on a current build, please comment and we will reopen.
